jQuery/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  margin: 20px 5%;
+}
+
+#hh-menu {
+  border-right: solid 1px black;
+  position: fixed;
+  width: 240px;
+  height: 100%;
+}
+
+#hh-menu > select {
+  width: 80%;
+  margin-left: 15%;
+  margin-right: 5%;
+}
+
+#hh-menu > label {
```

Comment:
Same as above, consider using a class for all your labels. direct child selectors are slower in older browsers (or sometimes unsupported entirely (in ie8))